### PR TITLE
Some ServerWorld, GameRenderer, and MinecraftServer mappings

### PR DESCRIPTION
**Pull request by alothmanmoe**
Just some methods I thought to map after using them.

### CODE REVIEW

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 1]
```diff
@@ -95,6 +95,7 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD a setResourcePack (Ljava/lang/String;Ljava/lang/String;)V
 		ARG 1 url
 	METHOD a setKeyPair (Ljava/security/KeyPair;)V
+	METHOD a update (Ljava/util/function/BooleanSupplier;)V
```
Comment (Runemoro) [inline mappings/net/minecraft/server/MinecraftServer.mapping]: How is this different from the `tick` method also in MinecraftServer?

Comment (alothmanmoe) [reply mappings/net/minecraft/server/MinecraftServer.mapping]: Whoops, nice catch, yeah it basically is just tick, I mapped it to update on accident while looking at another method I was going to map.

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 2]
```diff
@@ -95,6 +95,7 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD a setResourcePack (Ljava/lang/String;Ljava/lang/String;)V
 		ARG 1 url
 	METHOD a setKeyPair (Ljava/security/KeyPair;)V
+	METHOD a tick (Ljava/util/function/BooleanSupplier;)V
```
Comment (liach) [inline mappings/net/minecraft/server/MinecraftServer.mapping:98]: can you name the parameter?

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/server/MinecraftServer.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/MinecraftServer.mapping:98]: Wait, that's not what I meant... Now there's two methods with the same arguments in the same class named the same... 
Look below: `METHOD b tick (Ljava/util/function/BooleanSupplier;)V`

**Comment by liach** [comment]
Runemoro is right. These erroneous tick names definitely have issues.

**Comment by alothmanmoe** [comment]
Yup looking into it now, the method I just mapped with tick is being called by run() every loop, so honestly I'd think the name tick for that would be fine, however, the one currently mapped to tick is called by method_3748 (the one i mapped to tick) which by the looks of it is just updating time, entities, lighting. Maybe remap it to a more fitting name other than tick? Opinions on this?

**Comment by liach** [comment]
rename to describe the boolean suppliers.

**Comment by asiekierka** [comment]
The canAddEntity() one is not exactly right, and we'll probably get to the two other ones later.

**Pull request closed without merging**